foodWise/src/main/java/br/com/foodWise/foodWise/dto/UserDTO.java

```diff
@@ -0,0 +1,21 @@
+package br.com.foodWise.foodWise.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.time.LocalDateTime;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class UserDTO {
```

Comment:
Eu ainda estou modelando o banco então acredito que vamos ter que alterar a estrutura da model.

Reply:
Coloquei apenas a estrutura base para podermos trabalhar em cima. Se for necessário podemos deletar e iniciar uma nova classe depois.